src/Bolero.Server/Remoting.fs

```diff
@@ -114,28 +116,27 @@ type internal RemotingService(basePath: PathString, ty: System.Type, handler: ob
         let output = output.MakeGenericMethod(method.ReturnType)
         fun (auth: IAuthorizationService) (ctx: HttpContext) ->
             let run() =
-                let arg =
-                    using (new StreamReader(ctx.Request.Body)) Json.Raw.Read
-                    |> decoder
-                Remote.context.Value <- ctx
-                let res = callMeth.Invoke(meth, [|arg|])
-                output.Invoke(null, [|ctx; encoder; res|]) :?> Task
-            match tAuthPolicy with
-            | None -> run()
-            | Some tAuthPolicy ->
-                tAuthPolicy
-                    .ContinueWith(fun (tAuthPolicy: Task<AuthorizationPolicy>) ->
-                        auth.AuthorizeAsync(ctx.User, tAuthPolicy.Result)
-                    )
-                    .Unwrap()
-                    .ContinueWith(fun (tAuthResult: Task<AuthorizationResult>) ->
-                        if tAuthResult.Result.Succeeded then
-                            run()
-                        else
-                            fail ctx
-                            Task.CompletedTask
-                    )
-                    .Unwrap()
+                task {
```

Comment:
I see the switch from `async {}` to `task {}` in this pull request. Is that to line up with the upcoming F# task feature at https://github.com/dotnet/fsharp/pull/6634 ?

Reply:
Yeah, and also simply because it makes the code much nicer than a series of `.ContinueWith(...).Unwrap()`.